### PR TITLE
Breakpoint simplification

### PR DESCRIPTION
**Pull request by lemald**
Asana ticket: [⚙️ Finish up layout/breakpoint changes](https://app.asana.com/0/1148853526253420/1206526459943482/f)

### CODE REVIEW

File: assets/src/components/nav.tsx [hunk 1]
```diff
@@ -26,46 +23,30 @@ const Nav: React.FC<Props> = ({ children }) => {
...
     default:
       return (
         <div className="l-nav--wide">
           <>
-            <div className="l-nav__nav-bar l-nav__nav-bar--top">
+            <div
+              className="l-nav__nav-bar l-nav__nav-bar--top"
+              hidden={
+                isViewOpen && deviceType === "mobile_landscape_tablet_portrait"
+              }
```
Comment (firestack) [inline assets/src/components/nav.tsx:34]: question(non-blocking): I pulled this down locally and it worked fine, but it didn't work for me in dev green, any ideas why?

**Review by firestack — APPROVED** [review]

File: assets/src/components/nav.tsx (hunk 1 above)
Comment (lemald) [reply assets/src/components/nav.tsx:34]: Yeah I just ran into the same thing with design review, I'm pulling my hair out trying to figure it out.

Comment (lemald) [reply assets/src/components/nav.tsx:34]: Ah, the deploy action was marked as successful but the circuit breaker actually rolled it back. Gonna retry.

Comment (joshlarson) [reply assets/src/components/nav.tsx:34]: Wuttt... why would it do that?

Comment (lemald) [reply assets/src/components/nav.tsx:34]: I think it hit the ECS deploy circuit breaker that eventually gives up on failing deploys (it was having issues connecting to the DB). That's great and all but it isn't well-integrated into our deploy action so it basically sees that the deploy finished without detecting the particular error state it hit.

**Pull request merged**